python/srv

```diff
@@ -17,26 +17,27 @@
 import asyncio
 import sys
 import signal
-import veles.asrv.srv
-import veles.asrv.proto
 import importlib
 
+from veles.server.conn import AsyncLocalConnection
+from veles.server.proto import unix_server, tcp_server
+
 print('Świtezianka server starting up...')
 loop = asyncio.get_event_loop()
 print('Opening database.')
-srv = veles.asrv.srv.Server(loop, sys.argv[1])
+conn = AsyncLocalConnection(loop, sys.argv[1])
 print('Loading plugins.')
 for pname in sys.argv[3:]:
     print('{}...'.format(pname))
     mod = importlib.import_module('plugins.' + pname)
-    srv.add_local_plugin(mod.Plugin)
+    conn.add_local_plugin(mod.Plugin)
 host, _, port = sys.argv[2].rpartition(':')
 if host == 'UNIX':
     print('Starting UNIX server.')
-    loop.run_until_complete(veles.asrv.proto.unix_server(srv, port))
+    loop.run_until_complete(unix_server(conn, port))
 else:
     print('Starting TCP server.')
-    loop.run_until_complete(veles.asrv.proto.tcp_server(srv, host, int(port)))
+    loop.run_until_complete(tcp_server(conn, host, int(port)))
 print('Ready.')
 # loop.add_signal_handler(signal.SIGINT, loop.stop)
```

Comment:
I don't see any near future where this can be restored - the problem is that `add_signal_handler` isn't available on Windows so we can't use it :(
So lets remove it.